attest/ake/src/initiator.rs

```diff
@@ -124,22 +126,26 @@ where
         )
         .map_err(Error::HandshakeInit)?;
 
-        let mut serialized_report = Vec::with_capacity(input.ias_report.encoded_len());
-        input
-            .ias_report
-            .encode(&mut serialized_report)
-            .expect("Invariants failure, encoded_len insufficient to encode IAS report");
+        // TODO: replace with dcap
+        let verification_report = match input.attestation_evidence {
+            EvidenceKind::Epid(verification_report) => verification_report,
+            _ => Err(Error::AttestationEvidenceSerialization)?,
+        };
+        let mut serialized_evidence = Vec::with_capacity(verification_report.encoded_len());
+        verification_report
+            .encode(&mut serialized_evidence)
+            .map_err(|_| Error::AttestationEvidenceSerialization)?;
```

Comment:
This is probably due to my comment on EvidenceKind in its into/from bytes.
This is one of the "legacy" places I was thinking of
```suggestion
        let serialized_evidence = input.attestation_evidence.into_bytes();
```